src/typing_extensions.py

```diff
@@ -1105,9 +1171,14 @@ class Point2D(TypedDict):
                 "using the functional syntax, pass an empty dictionary, e.g. "
             ) + example + "."
             warnings.warn(deprecation_msg, DeprecationWarning, stacklevel=2)
-            if closed is not False and closed is not True:
+            # Support a field called "closed"
+            if closed is not False and closed is not True and closed is not None:
                 kwargs["closed"] = closed
-                closed = False
+                closed = None
+            # Or "extra_items"
+            if extra_items is not NoExtraItems:
+                kwargs["extra_items"] = extra_items
+                extra_items = NoExtraItems
```

Comment:
is this logic tested anywhere? I see a test here that might cover the `closed` logic, but I'm not sure I see one for the `extra_items` logic?

https://github.com/JelleZijlstra/typing_extensions/blob/4c20f9a62395b3d334e051b2854d5e17d82e1252/src/test_typing_extensions.py#L5032-L5041

Reply:
Good point, added a test case